source/_introduction.html.md.erb

```diff
@@ -298,9 +298,37 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibile option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
```

Comment:
backwards-compatible response*

also, is it important to point out that new column types will ALWAYS have a backwards compatible response? New grid data will never break someones API, no matter what level they are using.

Reply:
I can't verify this, but @jordyd might be able to. Are all level bumps (feature or new column type) backwards compatible? Have we made that commitment? 

I personally don't like the vagueness of the above statement. "In some cases... we offer a backwards compatible response". WHEN? WHY? WILL MY API's BREAK ALL THE TIME??

Reply:
Yeah, I think we basically want to convey that a level "bump" indicates the presence of a new feature that is NOT compatible with *existing* code, as it is inherently a new concept. In order to maintain compatibility with existing features, we introduce this concept so that a user who is currently requesting `someEndpoint?level=3` will not risk "breaking themselves" until/unless they bump to `someEndpoint?level=4`.

Reply:
If that comment doesn't make sense, I'm happy to discuss offline.

Reply:
That all makes sense. I'll figure out how to reword.

Reply:
I reworded this way:
When there is new functionality that is not compatible with existing code, say in the case of a new concept, Smartsheet uses a level "bump" to indicate the new feature can be ignored until you are ready to implement the code to work with the new level.

For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response: